es.json

```diff
@@ -390,6 +417,7 @@
   "paquete": 1,
   "pardillo": 2,
   "parguela": 2,
+  "pargo": 2,
```

Comment:
This might definitely be a locale thing, but isn't `pargo` just a fish? What's it referring to that could be offensive?

Reply:
I did find atleast 20+ geo/local words in the file that don't mean anything offensive to me also, are you a native spanish speaker?, as i said, these aditions are related to "Spanish as spoken in Venezuela". Btw. Pargo is a fish, but it is used as "marico"

Reply:
My coworker who helped me put this list of words together is a native Spanish speaker (Mexico City). I'm not a native speaker, but I do know enough to be functional :). I wouldn't say I'm the most fluent, but I know enough to get by. 

I did do a quick search and saw that `pargo` is (offensive) slang for homosexual, so I think it can stay.

Reply:
We do have tons of geo/local slangs in latin-america.
:)